.travis.yml

```diff
@@ -5,7 +5,15 @@ _environments:
   - &gcc-warnings
     - CFLAGS="-std=c89 -Wall -Wextra -pedantic"
 
+_scripts:
+  - &install_cmake35
+    - >
+        wget https://cmake.org/files/v3.5/cmake-3.5.2-Linux-x86_64.tar.gz &&
+        sudo tar xfC $(basename $_) / --strip-components=1
+
```

Comment:
Wow, good call all around. I see even now that the overall test has been reported, even as these stress-testers are still cranking away. Best of all worlds.